### PR TITLE
Add temporary eCredits survey to request send page

### PR DESCRIPTION
**Pull request by matharden**

### CODE REVIEW

**Comment by jasiek** [comment]
Looks good, provided this is temporary.

File: app/views/shared/_survey.html.haml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#user-satisfaction-survey-container
+  %section#user-satisfaction-survey
+    .wrapper
+      %h1 Help us design new ways of sending money to prisoners
+      %p.right
+        %a#survey-no-thanks{:href => "#survey-no-thanks"} No thanks
```
Comment (SteveMarshall) [inline app/views/shared/_survey.html.haml]: Should this be a `button` rather than an `anchor`?

**Comment by SteveMarshall** [comment]
I'm happy to :ship: this, but we need to check with @nicolekobilansky that the survey is ready to go…

**Comment by SteveMarshall** [comment]
:eggplant:!

**Pull request merged**